packages/moonstone/Scrollable/ScrollableNative.js

```diff
@@ -530,6 +594,56 @@ class ScrollableBaseNative extends Component {
 		this.uiRef.bounds.scrollHeight = this.uiRef.getScrollBounds().scrollHeight;
 	}
 
+	clearOverscrollEffect = (orientation, position) => {
+		const {type} = this.uiRef.getOverscrollStatus(orientation);
+
+		if (type !== overscrollTypes.none) {
+			this.overscrollJobs[orientation][position].startAfter(
+				(type === overscrollTypes.scrolling) ? overscrollTimeout : 0,
+				orientation,
+				position,
+				overscrollTypes.none,
+				0
+			);
+		}
+	}
+
+	clearAllOverscrollEffects = () => {
+		this.clearOverscrollEffect('horizontal', 'before');
+		this.clearOverscrollEffect('horizontal', 'after');
+		this.clearOverscrollEffect('vertical', 'before');
+		this.clearOverscrollEffect('vertical', 'after');
+	}
+
+	applyOverscrollEffect = (orientation, position, type, ratio) => {
+		const
+			nodeRef = this.overscrollRefs[orientation],
+			prefix = '--scrollable-overscroll-ratio-';
+
+		if (nodeRef) {
+			nodeRef.style.setProperty(prefix + orientation + position, ratio);
+
+			if (type === overscrollTypes.scrolling) {
+				this.overscrollJobs[orientation][position].start(orientation, position, overscrollTypes.none, 0);
+			}
+		}
+	}
+
+	newOverscrollJob = (orientation, position) => {
+		if (!this.overscrollJobs[orientation][position]) {
+			this.overscrollJobs[orientation][position] = new Job(this.applyOverscrollEffect.bind(this), overscrollTimeout);
```

Comment:
Do we need `bind` here because we defined it with arrow function for lexical scope?

Reply:
We need.